flixel/group/FlxGroup.hx

```diff
@@ -91,6 +96,9 @@ class FlxTypedGroup<T:FlxBasic> extends FlxBasic
 		members = [];
 		maxSize = Std.int(Math.abs(MaxSize));
 		flixelType = GROUP;
+		
+		addedToGroup = new FlxTypedSignal<T->Void>();
```

Comment:
I think it might be a good idea to initialize these lazily, to avoid unnecessary allocations for people who don't need these (which is probably most people).